apps.go

```diff
@@ -16,11 +16,11 @@ type AppLogType string
 
 const (
 	// AppLogTypeBuild represents build logs.
-	AppLogTypeBuild AppLogType = "build"
+	AppLogTypeBuild AppLogType = "BUILD"
```

Comment:
These need to up all caps due to the way protobuf (de)serializes json. I'm not sure if this is something we can change or make case insensitive. 🤔